gh/workflows: Bump CLI to v0.15.18

brb opened the pull request.

brb committed:
gh/workflows: Bump CLI to v0.15.18

Signed-off-by: Martynas Pumputis <m@lambda.lt>

sayboras reviewed (approved):
Thanks, hope new Ingress related tests will be fine 🙏

brb committed:
ci-e2e: Do not check for missed tail calls with host-fw

It's a known flake / bug [1].

[1]: https://github.com/cilium/cilium/issues/28088

Signed-off-by: Martynas Pumputis <m@lambda.lt>
